src/styles/partials/components/_breadcrumbs.scss

```diff
@@ -0,0 +1,73 @@
+.dg_breadcrumbs {
+  color: $black;
+  display: inline-block;
+  font-family: $default-heading-font;
+  font-size: $smallest-font-size;
+  font-weight: $font-bold;
+  list-style: none;
+  margin-bottom: 0px;
+  margin-top: $margin-small;
+  padding-bottom: $padding-smallest;
+  padding-left: 0px;
+  padding-top: 0px;
+  text-transform: uppercase;
+
+  i {
+    font-size: $h3-font-size;
+  }
+
+  ul {
+    margin-bottom: 0px;
+    padding-bottom: 0px;
+  }
+
+  li {
+    display: inline;
+
+    a {
+      color: $blue;
+      font-size: $smallest-font-size;
+      font-weight: $font-normal;
+      padding: 0px 30px 0px 0px;
+      text-decoration: none;
+
+      &:hover {
```

Comment:
you shouldn't need this, it will be inherited from the default link functionality (i think).

Reply:
Updated and committed.